app/models/metric/capture.rb

```diff
@@ -70,12 +70,12 @@ def self.perf_capture_timer(zone = nil)
       interval_name = self.perf_target_to_interval_name(target)
       next unless interval_name == "realtime"
 
-      parent = target.perf_rollup_parent(interval_name)
-      next(h) if parent.nil?
```

Comment:
So this check is now lost because the caller can still send a nil...[for example](https://github.com/ManageIQ/manageiq/pull/3044/files#diff-cae72f3c7d5b9ebc549497c6d8e9834aR1862) that could return `[nil]` if the host is orphaned and no longer connected to a cluster or ems.  Because the nil check here was removed that will propogate an error.  The perf_rollup_parents should return an empty Array if there aren't any parents.